README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://github.com/immutable-js/immutable-js/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/immutable-js/immutable-js/actions/workflows/ci.yml?query=branch%3Amain) [Chat on slack](https://immutable-js.slack.com)
 
-[Read the docs](https://immutable-js.com) and eat your vegetables.
+[Read the docs](https://immutable-js.com/docs/v4.3.4/) and eat your vegetables.
```

Comment:
```suggestion
[Read the docs](https://immutable-js.com/docs/) and eat your vegetables.
```

This way it will redirect automatically on the 5.x when available